Example/ChatWallet/ApplicationLayer/AppDelegate.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+@main
+final class AppDelegate: UIResponder, UIApplicationDelegate {
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        print("lala")
```

Comment:
can we remove it? xd

Reply:
Yeah, removed already, you shouldn't look there haha 😂